src/prefect/serialization/storage.py

```diff
@@ -18,12 +18,11 @@ class Meta:
     registry_url = fields.String(allow_none=True)
     image_name = fields.String(allow_none=True)
     image_tag = fields.String(allow_none=True)
-    flow_file_path = fields.String(allow_none=True)
```

Comment:
So are these going to be inferred now based on name?

Reply:
Nope, oops; I need to add this attribute to flow serialization.

Reply:
Addressed by serializing `flows` attribute on storage. 👍